### PR TITLE
Update windows-10-feature-updates.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about the end user update status as the last scanned only shows if the user logs in at least once.

### CODE REVIEW

**Comment by PRMerger16** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2965

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/protect/windows-10-feature-updates.md [hunk 1]
```diff
@@ -224,6 +224,10 @@ Selecting a profile from the list opens the profiles **Overview** pane where you
 - Select **Properties** to modify the deployment.  On the *Properties* pane, select **Edit** to open the *Deployment settings or Assignments*, where you can then modify the deployment.
 - Select **End user update status** to view information about the policy.
 
+
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/windows-10-feature-updates.md:227]:
```suggestion
```

File: memdocs/intune/protect/windows-10-feature-updates.md [hunk 2]
```diff
@@ -224,6 +224,10 @@ Selecting a profile from the list opens the profiles **Overview** pane where you
...
+> [!NOTE]
+> The End user update status Last Scanned Time value will return 'Not scanned yet' until an initial user logon and Update Session Orchestrator (USO) scan is initiated. For more information on the Unified Update Platform (UUP) architecture and related components, see (Get started with Windows Update)[https://docs.microsoft.com/windows/deployment/update/windows-update-overview].
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/windows-10-feature-updates.md:229]:
```suggestion
> The End user update status Last Scanned Time value will return 'Not scanned yet' until an initial user logs on and Update Session Orchestrator (USO) scan is initiated. For more information on the Unified Update Platform (UUP) architecture and related components, see (Get started with Windows Update)[https://docs.microsoft.com/windows/deployment/update/windows-update-overview].
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger19** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/protect/windows-10-feature-updates.md [hunk 3]
```diff
@@ -224,6 +224,9 @@ Selecting a profile from the list opens the profiles **Overview** pane where you
 - Select **Properties** to modify the deployment.  On the *Properties* pane, select **Edit** to open the *Deployment settings or Assignments*, where you can then modify the deployment.
 - Select **End user update status** to view information about the policy.
 
+> [!NOTE]
+> The End user update status Last Scanned Time value will return 'Not scanned yet' until an initial user logs on and Update Session Orchestrator (USO) scan is initiated. For more information on the Unified Update Platform (UUP) architecture and related components, see (Get started with Windows Update)[https://docs.microsoft.com/windows/deployment/update/windows-update-overview].
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/windows-10-feature-updates.md:228]:
```suggestion
> The End user update status Last Scanned Time value will return 'Not scanned yet' until an initial user logs on and Update Session Orchestrator (USO) scan is initiated. For more information on the Unified Update Platform (UUP) architecture and related components, see [Get started with Windows Update](/windows/deployment/update/windows-update-overview).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba, one more, please.

**Comment by PRMerger15** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Brenduns Please review and sign off this PR. Thanks

**Comment by Brenduns** [comment]
#sign-off 
Thanks!

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**